pkg/arvo/app/observe-hook.hoon

```diff
@@ -260,9 +288,53 @@
     ==  == 
   --
 ::
+++  on-arvo
+  |=  [=wire =sign-arvo]
+  ^-  (quip card _this)
+  ?+    wire  (on-arvo:def wire sign-arvo)
+      [%warm-cache ~]
+    ?.  warm-cache
+      [~ this]
+    ?>  ?=([%clay %writ *] sign-arvo)
+    :_  this
+    :-  =/  =rave:clay  [%next [%t da+now.bowl /mar]]
```

Comment:
It would be slightly more conceptually coherent to get the case from `q.p.u.riot.sign-arvo`, which will guarantee you get an update for every relevant revision with no gaps.  I'm not sure there's a case in the system as-is where this could break, but it relies on certain properties that we haven't always had and might not in the future.  A problematic case would be something along the lines of: another subscriber to clay hears this update first and makes another commit before this resubscription happens, so you don't hear about the second commit.  This wouldn't be a problem now because you're warming up the most recent commit no matter what (which is good), but it could be different if e.g. we adjust the meaning of a clay commit at `now.bowl`.

Reply:
Do you think I should stop using `da+now.bowl` in all of the scries, or just this one? I don't understand the semantics of the `case` and how it should be different well enough to be confident that this change will preserve the appropriate semantics.

Reply:
Just this one.  The others are properly thought of as "most recent", while this one is "next time this changes after the notification i got"

Reply:
What about the one on line 327 where we also send a `%next` request?